README.md

```diff
@@ -322,7 +322,7 @@ Promise.all(promises).whenComplete {
 implementation 'com.shopify.promises:promises:0.0.8'
 ```
 
-You can see the [lastest version in Bintray](https://bintray.com/shopify/shopify-android/Promises) 
+You can see previous releases on [Bintray](https://bintray.com/shopify/shopify-android/Promises)
```

Comment:
@gantonious Kept it but reworded. GitHub releases doesn't actually capture all previous releases so want it around for the full picture.